### PR TITLE
OSDOCS-8248: add kubeconfig refs to oc section

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13, 4.14, 4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-8248

Link to docs preview:
[About the OpenShift CLI](https://67104--docspreview.netlify.app/microshift/latest/microshift_cli_ref/microshift-cli-using-oc#microshift-cli-oc-about_microshift-using-oc)
[Additional resources](https://67104--docspreview.netlify.app/microshift/latest/microshift_cli_ref/microshift-cli-tools-introduction#additional-resources_microshift-cli-tools)

QE review:
- [x] QE has approved this change.

Manual CP to 4.13: https://github.com/openshift/openshift-docs/pull/67286

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @ShaunaDiaz,
    Had one minor suggestion, otherwise LGTM!

File: modules/microshift-cli-oc-about.adoc [hunk 1]
```diff
@@ -11,3 +11,8 @@ With the OpenShift command-line interface (CLI), the `oc` command, you can deplo
 * Working directly with project source code
 * Scripting {product-title} operations
 * Managing projects while restricted by bandwidth resources
+
+[NOTE]
+====
+A `kubeconfig` file must exist for the cluster to be accessible. The values are applied from built-in default values or a `config.yaml`, if one was created.
```
Comment (dhensel-rh) [inline modules/microshift-cli-oc-about.adoc:17]: I have a misunderstanding of kubeconfig. Can I modify the contents of kubeconfig by making changes in config.yaml ?

Comment (ShaunaDiaz) [reply modules/microshift-cli-oc-about.adoc:17]: @dhensel-rh The application creates them at startup from the yaml, yes. https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.14/html/configuring/microshift-kubeconfig

Comment (dhensel-rh) [reply modules/microshift-cli-oc-about.adoc:17]: ok

**Comment by bergerhoffer** [comment]
@ShaunaDiaz I don't see the 4.13 CP for this PR. Can you please either do the manual CP for it (or if you have done it already, please update the description to include a link to this PR (use this syntax: `#67104`) so that it is trackable from this original PR)?

**Pull request merged**